codec/reflectcodec/type_codec.go

```diff
@@ -85,12 +85,14 @@ func (c *genericCodec) Size(value interface{}) (int, error) {
 		return 0, errMarshalNil // can't marshal nil
 	}
 
-	size, _, err := c.size(reflect.ValueOf(value))
+	size, _, err := c.size(reflect.ValueOf(value), false)
```

Comment:
Personally I like to put comments to avoid "naked" literal arguments
```suggestion
	size, _, err := c.size(reflect.ValueOf(value), false /*nullable*/)
```